src/atoms/hooks/useSelect.ts

```diff
@@ -207,6 +207,9 @@ const useSelect = <T extends SelectOptionRequired>(
       search === ''
     ) {
       setTryingToRemoveItem(selectedValues?.at(-1));
+      setTimeout(() => {
+        searchRef.current?.focus();
+      });
```

Comment:
What does this setTimeout do? 🤔

Reply:
It's a hack so the searchRef get's focused after the setState above it has been called 😄 Same as in the handleOnSearchChange function

Reply:
Ah :D Tricksy 😏